QUDAConfig.cmake.in

```diff
@@ -6,14 +6,31 @@ set(QUDA_QMP @QUDA_QMP@)
 set(QUDA_MPI @QUDA_MPI@)
```

Comment:
Just a thought I want to put down here so I don't forget: Should we move that to the `lib` dir? After all this file describes the usage requirements for libquda . Might also make it easier with the `QUDA_TARGET_XYZ` definitions.